migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,276 @@
+18  +++   <kogito.bom.group-id>com.ibm.bamoe</kogito.bom.group-id>
+19  +++   <kogito.bom.artifact-id>bamoe-bom</kogito.bom.artifact-id>
+20  +++   <kogito.bom.version>{BOM_VERSION}</kogito.bom.version>
+21      </properties>
+----
+
+[start=4]
+. Replace lines 29-31, which is essentially just adding the `bamoe-bom` to the list of imported dependencies:
+
++
+[source,xml]
+----
+28      <dependency>
+29  ---   <groupId>${quarkus.platform.group-id}</groupId>
+30  ---   <artifactId>quarkus-kogito-bom</artifactId>
+31  ---   <version>${quarkus.platform.version}</version>
+29  +++   <groupId>${kogito.bom.group-id}</groupId>
+30  +++   <artifactId>${kogito.bom.artifact-id}</artifactId>
+31  +++   <version>${kogito.bom.version}</version>
+32        <type>pom</type>
+33        <scope>import</scope>
+34      </dependency>
+----
++
+A bill of materials (BOM) is a list of the parts or components that are required to build a product.
+BOMs are ordinary `pom.xml` files — they contain no source code and their only purpose is to declare their bundled modules.
+
+.Always use the Maven Bill-of-Materials (BOM)
+[IMPORTANT]
+====
+It is highly recommended to use a bill of materials (BOM) to manage project depencencies, as it simplifies the dependency management by organizing a group of depdenencies under a single version. 
+
+It should be noted that the use of Maven bill-of-materials (BOM) does not add any actual dependencies to the project, they are only definition that will help resolve properly (versions) the dependendies (and transient dependencies).
+====
+
+A bill of materails (BOM) can be added to an existing POM file by adding it to the **_dependencyManagement_** section as a dependency with a pom type:
+
+include::../partials/maven-bom-now.adoc[]
+
+When a bill of materials (BOM) is used it is no longer necessary to specify the depenency's version, as that comes from the BOM, so your depenencies end up looking like this:
+
+[source,xml]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+    </dependency>
+</dependencies>
+----
+
+[NOTE]
+====
+If, during the course of your Maven dependency update, you encounter any dependencies that are *not* included in the bill of materials (BOM), then use the Maven Library Version noted in the table below, such as:
+
+[source,xml,subs="attributes+"]
+----
+<dependencies>
+    <dependency>
+        <groupId>org.drools</groupId>
+        <artifactId>drools-compiler</artifactId>
+        <version>{BOM_VERSION_OLD}</version>
+    </dependency>
+</dependencies>
+----
+
+You should also report to IBM that a dependency is missing so that the problem will be resolved in next release.
+====
+
+The following table is useful in determining which community release is bound to is section details the mapping of https://www.kie.org/[KIE Community] releases to the current {PRODUCT_LONG} enterprise release.
```

Comment:
bound to is section details the mapping -> Note clear, may be you need to re-write this sentence